.github/workflows/publish.yml

```diff
@@ -17,6 +17,6 @@ jobs:
       env:
         CRATES_IO_TOKEN: ${{ secrets.crates_io_token }} # https://help.github.com/en/articles/virtual-environments-for-github-actions#creating-and-using-secrets-encrypted-variables
     - name: Dry run publish
-      run: cargo publish --dry-run
+      run: cd cloudflare && cargo publish --dry-run
```

Comment:
does this dir change persist to the next line?

Reply:
Hopefully. Broke it into its own line instead.